metalearn/metafeatures/static_assets.py

```diff
@@ -0,0 +1,6 @@
+import pkg_resources
+
+# Expose the `metafeatures_schema.json` file as a file path string.
+# Needed to allow the file to be successfully accessed across platforms 
+# and install types.
+METAFEATURES_JSON_SCHEMA = pkg_resources.resource_filename('metalearn', 'metafeatures/metafeatures_schema.json')
```

Comment:
Is this the schema or the path to the schema?

Reply:
That is the path to the schema. This line is the same as it was in our most recent release of this package.